crates/wallet/tests/wallet.rs

```diff
@@ -3997,6 +4006,41 @@ fn test_taproot_load_descriptor_duplicated_keys() {
     );
 }
 
+// In dev mode this test panics, but in release mode, or if the debug_panic is commented out,
+// there is no panic and the balance is calculated correctly. See #1483 and #1486 for discussion
+// on mixing non-wildcard and wildcard descriptors.
+#[test]
+#[should_panic(
+    expected = "replenish lookahead: must not have existing spk: keychain=Internal, lookahead=25, next_store_index=0, next_reveal_index=0"
+)]
+fn test_keychains_with_overlapping_spks() {
```

Comment:
```suggestion
#[test]
#[should_panic(
    expected = "replenish lookahead: must not have existing spk: keychain=Internal, lookahead=25, next_store_index=0, next_reveal_index=0"
)]
/// In `dev` mode this test panics, but in `release` mode, or if the `debug_panic` is commented out,
/// there is no panic and the balance is calculated correctly. See issues #1483 and #1486 for discussion
/// on mixing non-wildcard and wildcard descriptors.
fn test_keychains_with_overlapping_spks() {
```

The function below this one has docstrings.
Maybe these can be docstrings as well?
I also added the `issue` identifier to 1483 and 1486.

Reply:
I fixed up the docs as suggested. Even though these docs won't show up in the official rust docs it's still nice to see them highlighted and linked, at least in my IDE.